### ISSUE TITLE
Example notebook bug?

### ISSUE THREAD

**Issue by kvenkman** [kornia/tutorials#86]
I was looking at the [data augmentation example notebook](https://github.com/kornia/tutorials/blob/4789279aec8c3a88711560e689e28042d94cd57d/nbs/data_augmentation_kornia_lightning.ipynb), and was hoping to clarify a potential bug. In the `training_step` method:

```
    def training_step(self, batch, batch_idx):
        # REQUIRED
        x, y = batch
        x_aug = self.transform(x)  # => we perform GPU/Batched data augmentation
        logits = self.forward(x_aug)
        loss = F.cross_entropy(logits, y)
        self.log("train_acc_step", self.accuracy(logits.argmax(1), y))
        self.log("train_loss", loss)
        return loss
```
the input tensor `x` gets transformed to `x_aug`, yet the corresponding transform does not get applied to the labels `y` before loss is calculated (`loss = F.cross_entropy(logits, y)`). Is this an error that needs fixing, or am I missing something?

**Comment by edgarriba** [comment]
In this example we follow a standard augmentation strategy applying perturbations only in the input image. Labels are image unique ids, which do not get modified by color jitter augmentation

**Comment by kvenkman** [comment]
Oh, so the labels are one hot encoded vectors?

**Comment by kvenkman** [comment]
I see that now - thank you for the clarification!
